src/umpire/Allocator.cpp

```diff
@@ -6,91 +6,80 @@
 //////////////////////////////////////////////////////////////////////////////
 #include "umpire/Allocator.hpp"
 
+#include "umpire/Replay.hpp"
 #include "umpire/ResourceManager.hpp"
 #include "umpire/util/Macros.hpp"
-#include "umpire/Replay.hpp"
 
 #if defined(UMPIRE_ENABLE_STATISTICS)
-#include "umpire/util/StatisticsDatabase.hpp"
 #include "umpire/util/Statistic.hpp"
+#include "umpire/util/StatisticsDatabase.hpp"
 #endif
 
 namespace umpire {
 
-Allocator::Allocator(strategy::AllocationStrategy* allocator) noexcept:
-  m_allocator(allocator)
+Allocator::Allocator(strategy::AllocationStrategy* allocator) noexcept
+    : m_allocator(allocator)
 {
 }
 
-void
-Allocator::release()
+void Allocator::release()
 {
-  UMPIRE_REPLAY("\"event\": \"release\", \"payload\": { \"allocator_ref\": \"" <<  m_allocator << "\" }");
+  UMPIRE_REPLAY("\"event\": \"release\", \"payload\": { \"allocator_ref\": \""
+                << m_allocator << "\" }");
 
   UMPIRE_LOG(Debug, "");
 
   m_allocator->release();
 }
 
-std::size_t
-Allocator::getSize(void* ptr) const
+std::size_t Allocator::getSize(void* ptr) const
```

Comment:
I actually prefer this style of keeping the method on the same line as the return type when possible.

Reply:
Haha this is the one thing I don't really like 😄